lib/pages/login.dart

```diff
@@ -17,12 +22,60 @@ class LoginPage extends StatefulWidget {
 
 class _LoginPageState extends State<LoginPage> {
   bool loading = false;
+  Completer<WebViewController> _controller = Completer<WebViewController>();
   @override
   Widget build(BuildContext context) {
     SizeConfig().init(context);
     return Scaffold(
       body: loading
-          ? const PowerUp()
+          ? WebView(

```

Comment:
Try adding a SafeArea() here. The logo almost touches the status bar on my device.